src/hintingtt.js

```diff
@@ -79,14 +79,14 @@ function roundToGrid(v) {
 * Rounding to double grid.
 */
 function roundToDoubleGrid(v) {
-    return Math.sign(v) * Math.round(Math.abs(v / 2)) * 2;
+    return Math.sign(v) * Math.round(Math.abs(v * 2)) / 2;
```

Comment:
"In TrueType, rounding is symmetric about zero" - https://developer.apple.com/fonts/TrueType-Reference-Manual/RM05/Chap5.html

For example -7.5 is supposed to round to -8, not -7 as Math.round(-7.5) would yield